tools/tertiary-analysis/sc3/sc3-sc3-calc-biology.xml

```diff
@@ -1,52 +1,30 @@
-<tool id="sc3-sc3-calc-biology" name="SC3 calculate biology" version="0.1.0">
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="sc3-sc3-calc-biology" name="SC3 calculate biology" version="@TOOL_VERSION@+galaxy0">
     <description>SC3 calculates biologically relevant clusters</description>
     <macros>
-        <import>macros.xml</import>
+        <import>sc3-macros.xml</import>
     </macros>
     <expand macro="requirements" />
     <command detect_errors="exit_code"><![CDATA[
         sc3-sc3-calc-biology.R -i "$input1" -o "$output1"
     ]]></command>
     <inputs>
-        <param type="data" name="input1" format="rds" />
+        <param type="data" name="input1" format="rdata" label="File name in which a SC3 'SingleCellExperiment' object has been stored after kmeans clustering"/>
     </inputs>
     <outputs>
-        <data name="output1" format="rds" />
+        <data name="output1" format="rdata" label="File name for R object of type 'SingleCellExperiment' from SC3 in which to store the consensus matrix"/>
```

Comment:
The label of outputs will be displayed in the history, so it would be good to differentiate the entries if there are multiple runs. A commonly used label is `"${tool.name} on ${on_string}: some explanation"`. `${on_string}` will show user the result is based on which input datasets.
Please also check outputs of other wrappers.

Reply:
Hi @nh3, can you point me to an example in your scanpy wrappers? Thanks!

Reply:
Never mind @nh3, I found it!